### PR TITLE
Update cosmos sample from track1 to track2(Developing a Java app usin…

### PR DESCRIPTION
**Pull request by Melanie-Ma**
Update cosmos sample from track1 to track2(Developing a Java app using Azure Cosmos DB Async Java SDK)

@jongio for notification

### CODE REVIEW

File: azure-cosmosdb-get-started/pom.xml [hunk 1]
```diff
@@ -41,9 +41,9 @@
   </build>
   <dependencies>
     <dependency>
-      <groupId>com.microsoft.azure</groupId>
-      <artifactId>azure-cosmosdb</artifactId>
-      <version>2.2.0</version>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-cosmos</artifactId>
+      <version>4.0.0-preview.1</version>
```
Comment (jongio) [inline azure-cosmosdb-get-started/pom.xml]: Please use the latest GA version.

**Review by jongio — CHANGES_REQUESTED** [review]

**Comment by jongio** [comment]
@kushagraThapar - Can you please review?

**Comment by kushagraThapar** [comment]
@jongio - I have the same opinion on this getting started guide as I did for the samples. We already have azure-cosmos-java-getting-started guide based on track 2 library cosmos V4 SDK here - https://github.com/Azure-Samples/azure-cosmos-java-getting-started

I think we should just redirect this to the existing one. Thoughts ?

**Comment by jongio** [comment]
@Melanie-Ma - Can you please update this one to point to the new one?

@tzhanl

**Comment by Melanie-Ma** [comment]
@jongio please review.

**Review by jongio — APPROVED** [review]

**Review by kushagraThapar — APPROVED** [review]
LGTM

**Pull request merged**